src/modules/30-secrets/components/CreateUpdateSecret/views/VaultFormFields.tsx

```diff
@@ -64,6 +87,19 @@ const VaultFormFields: React.FC<VaultFormFieldsProps & FormikContextProps<any>>
       {type === 'SecretFile' ? (
         <FormInput.FileInput name="file" label={getString('secrets.secret.labelSecretFile')} multiple />
       ) : null}
+      {secretManagerType === 'GcpSecretManager' &&
+        (formik?.values['valueType'] === 'Reference' ? (
+          <>
+            <FormInput.Text name="version" label={getString('version')} />
+          </>
+        ) : (
+          <>
+            <FormInput.CheckBox name="configureRegions" label={getString('secrets.secret.configureRegion')} />
+            {formik?.values['configureRegions'] ? (
```

Comment:
You can probably just use the standard `.` notation instead of using the array access notation
```suggestion
            {formik?.values?.configureRegions ? (
```